src/integrators/integrator_utils.jl

```diff
@@ -198,6 +199,23 @@ function step_reject_controller!(integrator,alg::JVODE)
   integrator.dt *= integrator.qold
 end
 
+function stepsize_controller!(integrator,alg::RichardsonEuler)
```

Comment:
this is just a standard proportional controller? If so, no reason to just copy paste the code. Just change how it dispatches.

Reply:
@ChrisRackauckas Here controller modifies `dt` according to calculated `dtpropose` in `perform_step`.

Reply:
ahh it changed.